vendor/bat-native-ledger/src/bat/ledger/internal/media/reddit.cc

```diff
@@ -282,4 +282,91 @@ std::string Reddit::GetProfileImageUrl(const std::string& response) {
   return image_url;  // old reddit does not use account icons
 }
 
+void Reddit::OnMediaPublisherInfo(
+    const std::string& user_name,
+    ledger::PublisherInfoCallback callback,
+    ledger::Result result,
+    ledger::PublisherInfoPtr publisher_info) {
+  if (result != ledger::Result::LEDGER_OK &&
+      result != ledger::Result::NOT_FOUND) {
+    callback(ledger::Result::LEDGER_ERROR, nullptr);
+    return;
+  }
+  GURL url(REDDIT_USER_URL + ledger_->URIEncode(user_name));
+  if (!url.is_valid()) {
+    callback(ledger::Result::TIP_ERROR, std::move(publisher_info));
```

Comment:
should we return info here or just null if we have an error?

Reply:
based on other callbacks with errors I would say just return nullptr

Reply:
done